lib/backend/k8s/resources/watcher.go

```diff
@@ -104,16 +104,9 @@ func (crw *k8sWatcherConverter) processK8sEvents() {
 		case event, ok := <-crw.k8sWatch.ResultChan():
 			var events []*api.WatchEvent
 			if !ok {
-				// The channel is closed so send a terminating watcher event indicating the watch was
-				// closed by the remote.
-				crw.logCxt.Debug("Watcher terminated by remote")
-				events = []*api.WatchEvent{{
-					Type: api.WatchError,
-					Error: cerrors.ErrorWatchTerminated{
-						Err:            fmt.Errorf("terminating error event from Kubernetes watcher: closed by remote"),
-						ClosedByRemote: true,
-					},
-				}}
+				//Watch channel is closed by remote k8s.hence, return from loop.
```

Comment:
nit:  space after //

Reply:
Taken care. Updated.

Reply:
Still seems to be a problem in the PR :-)

Reply:
typo:  k8s.hence

Reply:
Taken care both cases.